fr.html

```diff
@@ -2034,64 +2041,64 @@
 <span id="label_chapter_links">
 	<a href="#0">Intro</a>
 	&middot;
-	<a href="#1">The Science</a>
+	<a href="#1">La Science</a>
 	&middot;
-	<a href="#2">The Art</a>
+	<a href="#2">L'Art</a>
 	&middot;
-	<a href="#3">Get Started!</a>
+	<a href="#3">Débutons !</a>
 	&middot;
-	<a href="#bye">Credits</a>
+	<a href="#bye">Crédits</a>
 </span>
 
 <!-- Multi Card Labels -->
 <span id="multicard_q">
-	try to recall &uarr;
-	then flip ↻
+	essaie de te rappeler
+	puis tourne ↻
 </span>
 <span id="multicard_cards_left">
-	(cards left: [N])
+	(cartes restantes: [N])
 </span>
 <span id="multicard_a">
-	did you remember this?
+	tu t'en souvenais ?
 </span>
 <span id="multicard_no">
-	nah, try again
+	non, on réessaie
 </span>
 <span id="multicard_yes">
-	yup, onwards!
+	oui, on continue !
 </span>
 <span id="multicard_done">
-	done for now! keep scrolling
+	c'est bon ! continue de scroller
 	<br>
 	&darr;
 </span>
 <span id="multicard_done_2">
-	that's all, folks!
+	c'est terminé !
 	<br>
 	&darr;
 </span>
 
 <!-- Ebbinghaus Simulation -->
 <span id="ebbinghaus_y_axis">
-	strength of memory &rarr;
+	vivacité du souvenir &rarr;
```

Comment:
le dernier curseur est à moitié affiché

Reply:
I changed the CSS a bit, I'll see if it was corrected when deployed.